client/components/theme-tier/theme-tier-badge/theme-tier-community-badge.js

```diff
@@ -45,15 +41,28 @@ export default function ThemeTierCommunityBadge() {
 
 	return (
 		<>
-			<ThemeTierBadgeTracker />
+			{ ! legacyCanUseTheme && (
+				<>
+					<ThemeTierBadgeTracker />
+					<PremiumBadge
```

Comment:
[Non-blocking]
Maybe we shouldn't call this `PremiumBadge` anymore. Right now when I'm on a Business plan and I look at a Community theme thumbnail, a PremiumBadge is being rendered with the label `Community`. Maybe this component should be something like `ThemeBadge` instead, what do you think?

Reply:
Agreed! It's already a misnomer TBH (e.g. it's used for Partner and Community labels).

We can rename the component separately, in the cleanup phase. 🙂 
I've created a new tasklist on https://github.com/Automattic/dotcom-forge/issues/4597. Let's add these tasks there so that we don't forget about them!